template-bff-service/test/int/mocks.js

```diff
@@ -1,5 +1,22 @@
 // eslint-disable-next-line no-unused-vars
 import * as sinon from 'sinon';
 import * as utils from '../../src/utils';
+import Replay from 'replay';
 
+// ==========================
+// Mock dates, uuids, etc
+// ==========================
 sinon.stub(utils, 'now').returns(1653877763001); // TODO update when re-recording
+
+// ==========================
+// VCR - https://github.com/assaf/node-replay
+// ==========================
+Replay.mode = process.env.REPLAY;
+console.log(`Replay mode = ${Replay.mode}`);
+
+// typical headers
+Replay.headers = [/^x-amz-target/].concat(Replay.headers.filter((header) => !header.test('authorization') && !header.test('x-')));
+
+// monkey patch replay to avoid immediate max retries and ultimately timeout
+const rproxy = require('replay/lib/proxy');
+rproxy.prototype.setTimeout = () => { };
```

Comment:
- serverless-offline now starts the api gateway and lambda endpoints in separate processes from the serverless framework
- but the baton vcr plugin runs in the sls process
- so we need to include Replay in the webpack bundle so that we initialize it in the right processes